### PR TITLE
Contact page updated as per requirement

### PR DESCRIPTION
**Pull request by Abdullah-Irfan-web**
Changes the Contact page as per the instruction

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
   	<link rel="stylesheet" href="css/vegas.min.css">
     <link rel="stylesheet" href="css/navstyle.css">
 	<link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/845b7f6704.js" crossorigin="anonymous"></script>
```
Comment (pankajan05) [inline contact.html:39]: This script need to remove

File: contact.html [hunk 2]
```diff
@@ -139,56 +141,53 @@
...
             <div class="col-sm-12 m-auto">
                 <!-- Page Title -->
-                <h1>Contact Us</h1>
+                <h1>Get Touch With Us</h1>
+                <div  style="background-color: red; height: 4px; width: 55px; margin: auto;" class="line">
+
+                </div>
                 <!-- Page Description -->
-                <p> Please contact us with any questions regarding our irrigation system. We're here to help via email, chat, or text.</p>
+                <p style="margin-top: 25px;"> Need to speak to a leading bespoke software development company? We will respond to all your enquires sent to the email address below and via our contact us form within 1 working day </p>
```
Comment (pankajan05) [inline contact.html:149]: The content needs to change according to Elzian agro that provide smart agronomy solution.

File: contact.html [hunk 3]
```diff
@@ -139,56 +141,53 @@
...
-            <div class="col-lg-7 mt-5 mt-lg-0 mc-auto">
-                <div class="google-map">
-                    <!-- Google Map -->
-                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d990.0694320298157!2d79.91026292921171!3d6.976522399685001!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3ae25908209d5e03%3A0x60f3a8686d7e401b!2sElzian%20Pvt%20Ltd!5e0!3m2!1sen!2slk!4v1631353723480!5m2!1sen!2slk" width="100%" height="450" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
-                </div>
+            <div style="flex: 0.3; border: 2px solid rgb(238, 237, 237); box-shadow: 4px 4px 2px rgb(238, 237, 237); margin-left: 30px;  display: flex; flex-direction: column; align-items: center; justify-content: center;padding: 30px 10px;" >
+                <i style="font-size: 50px; color: rgb(238, 57, 57);;" class="far fa-clock"></i>
+
+                <span style="padding-top: 30px; color: black;font-weight: 500; font-size: 20px; ">Office Schedule</span>
+                <div style="background-color: rgb(233, 231, 231); height: 2px; width:47px;"></div>
```
Comment (pankajan05) [inline contact.html:172]: The color must be the green color that similar to the logo's green.

File: contact.html [hunk 4]
```diff
@@ -139,56 +141,53 @@
...
-                        </div>
-                        <div class="details">
-                            <p>ELZIAN (PVT) LTD,<br>
-                                124/1 ,
-                                Mahawaththa Rd,<br>
-                                Kelaniya, Sri Lanka. </p>
-                        </div>
-                    </div>
-                </div>
+<section class="addres">
```
Comment (pankajan05) [inline contact.html:155]: section padding need to add at the top and bottom.

File: contact.html [hunk 5]
```diff
@@ -139,56 +141,53 @@
...
+    <div class="containe">
+        <div style="display: flex;" class="">
+            <div style="flex: 0.3; border: 2px solid rgb(238, 237, 237); box-shadow: 4px 4px 2px rgb(238, 237, 237); margin-left: 30px; display: flex; flex-direction: column; align-items: center; justify-content: center;padding: 30px 10px;" >
+
+                <i style="font-size: 48px; color: rgb(238, 57, 57);" class="fas fa-map-marked-alt"></i>
```
Comment (pankajan05) [inline contact.html:160]: inline styles needed to remove and added to the style.css

**Review by pankajan05 — COMMENTED** [review]
Good work @Abdullah-Irfan-web

**Comment by pankajan05** [comment]
Contact pankajan@elzian.com or lilan@elzian.com

File: contact.html (hunk 5 above)
Comment (Abdullah-Irfan-web) [reply contact.html:160]: @pankajan05 will resolve it asap

File: contact.html (hunk 3 above)
Comment (Abdullah-Irfan-web) [reply contact.html:172]: @pankajan05 are you talking about the i tag? They were like red that was given in Issue section

File: contact.html (hunk 4 above)
Comment (Abdullah-Irfan-web) [reply contact.html:155]: @pankajan05 OK got it

File: contact.html (hunk 1 above)
Comment (Abdullah-Irfan-web) [reply contact.html:39]: @pankajan05 this is for fontawesom icons. Should I use images instead of icons?

File: contact.html (hunk 3 above)
Comment (pankajan05) [reply contact.html:172]: No that is design purpose we don't need the red we need that in green color

Comment (Abdullah-Irfan-web) [reply contact.html:172]: Ok I will fix it

File: contact.html (hunk 1 above)
Comment (pankajan05) [reply contact.html:39]: Yeah, that is fine because it will create unnecessary network load. and you can download the icons and you can use them.

Comment (Abdullah-Irfan-web) [reply contact.html:39]: Ok got it. All the issues will be resolved by today only

**Comment by Abdullah-Irfan-web** [comment]
i have updated the changes as you asked

**Comment by pankajan05** [comment]
Great I will check those changes. Thank you irfan.

**Comment by pankajan05** [comment]
The contact us section has mobile responsibility issues please use the bootstrap columns.

**Pull request merged**